website/src/pages/admin/AllDevices.tsx

```diff
@@ -43,7 +43,7 @@ export const AllDevices = observer(class AllDevices extends React.Component {
   }
 
   deleteDevice = async (device: Device.AsObject) => {
-    if (await confirm('Are you sure?')) {
+    if (await confirm('Delete ' + device.name + ' from ' + device.ownerName + ' are you sure?')) {
```

Comment:
Are you sure you want to delete ' + device.name + ' from ' + device.ownerName + '?'

Reply:
![image](https://github.com/freifunkMUC/wg-access-server/assets/5702338/2a5f7488-2061-4e47-9524-58406e43d21e)